docs/src/development/cli/_index.md

```diff
@@ -1,148 +1,161 @@
+### Select the right project and environment
 
-## Auto-selecting your project
-
-When your shell's working directory is inside a local checkout of your project repository, the CLI will automatically detect your project ID and environment so you don't need to list them as parameters each time.
-
-In your home directory, for example, you need to provide the project ID as an argument each time:
+When you are in an empty directory or a directory not associated with a specific Platform.sh project,
+if you run a command that requires a specific project and environment, you are prompted to select them.
 
 ```bash
-$ platform project:info --project=acdefghijkl --environment=staging
+$ platform environment:info
+Enter a number to choose a project:
+  [0] My project (xb3pfo734qxbeg)
+  [1] A great project (3p5fmol45kxp6)
+  [2] An even better project (rjify4y564xaa)
+> 
 ```
 
-You can instead get the same result with just:
+If your working directory is inside a local checkout of your project repository,
+your project and environment are detected automatically.
 
-```bash
-$ cd myproject
-$ platform project:info
-```
+You can always specify the project and environment in two ways:
 
-You can also set a preferred project ID with the environment variables `PLATFORM_PROJECT`, `PLATFORM_BRANCH` and `PLATFORM_APPLICATION_NAME`.
+* As arguments for the command:
 
-```bash
-export PLATFORM_PROJECT=acdefghijkl;
-export PLATFORM_BRANCH=staging;
-platform project:info
-```
+  ```bash
+  $ platform environment:info --project=my-project --environment=staging
+  ```
+* With environment variables:
 
-## Autocomplete on the command line
+  ```bash
+  export PLATFORM_PROJECT=my-project;
+  export PLATFORM_BRANCH=staging;
+  platform environment:info
+  ```
 
-Once installed, the `platform` CLI tool provides tab auto-completion for commands, options, and even some values (your projects, valid regions).
+In [multi-app](../../configuration/app/multi-app.md) projects, this applies also to selecting the right app
+(the environment variable would be `PLATFORM_APPLICATION_NAME`).
 
-{{< note >}}
-Your system must include the `bash-completion` package or equivalent. This is not available by default on macOS, but can be installed via `brew`. Check your home directory and ensure that the file `~/.platformsh/autocompletion.sh` is being included by your shell. `platform self:install` will attempt a reinstall of this utility if it's needed.
-{{< /note >}}
+### Choose between the CLI and Git commands
 
-## Installing the CLI on Windows 10
+Some CLI commands (especially many within the `environment` namespace) have some overlap with Git commands.
+Generally, they offer more options than the Git commands alone.
+For example, `platform push` offers options such as `--activate` (to active an environment before pushing)
+and `--no-wait` (so you can continue working without waiting for the push to complete).
```

Comment:
The Google Style Guide recommends keeping parentheses to a minimum. I personally try to avoid them when possible just because I feel they cut the rhythm. I could keep the one in line 134 but I would remove those in 136 and 137. Up to you

Reply:
The Google style guide recommends against parentheses for important information. This information isn't important.

It suggests considering commas (which don't work well in this case), dashed (which would mean em dashes, which I dislike), semicolons (which don't work at all here), and periods (which don't work here).

So I think the parentheses work OK. The ones in 136 & 137 serve the same purpose as those in 134.